### ISSUE TITLE
The plugin does not supported on Grafana 8

### ISSUE THREAD

**Issue by shais-mmg** [doitintl/bigquery-grafana#341]
## Bug Report

After upgrading the Grafana 8 the plugin stopped working and it seems like it is not supported on Grafana 8.
It seems to work fine up to Grafana version 7.5.7

![image](https://user-images.githubusercontent.com/51051387/121353102-2b7d9700-c936-11eb-86ed-6f17c1e8e3ba.png)

![image](https://user-images.githubusercontent.com/51051387/121353320-67b0f780-c936-11eb-9ff9-f4ba4ae751e7.png)

**Comment by JorritSalverda** [comment]
I have the same. Even with environment variables

```
GF_INSTALL_PLUGINS: 'doitintl-bigquery-datasource'
GF_ALLOW_LOADING_UNSIGNED_PLUGINS: 'doitintl-bigquery-datasource'
```

I get the following logs:

```
✔ Downloaded doitintl-bigquery-datasource v1.0.8 zip successfully
lvl=warn msg="Some plugin scanning errors were found" logger=plugins errors="plugin 'doitintl-bigquery-datasource' is unsigned"
lvl=info msg="inserting datasource from configuration " logger=provisioning.datasources name="Google BigQuery" uid=
lvl=eror msg="Could not find plugin definition for data source: doitintl-bigquery-datasource"
```

Haven't found a way to fix it yet.

**Comment by JorritSalverda** [comment]
The plugin configurations page shows the following message.

![unsigned-plugin](https://user-images.githubusercontent.com/136472/121365815-430a4f00-c939-11eb-923e-3a8e0b6b2085.png)

And links to page https://grafana.com/docs/grafana/latest/plugins/plugin-signatures/. So perhaps a new release with a signature needs to be created?

**Comment by JorritSalverda** [comment]
Turns out I used the wrong envvar for allowing unsigned plugins. Settings

```
GF_PLUGINS_ALLOW_LOADING_UNSIGNED_PLUGINS: 'doitintl-bigquery-datasource'
```

It then logs

```
✔ Downloaded doitintl-bigquery-datasource v1.0.8 zip successfully
t=2021-06-09T13:45:17+0000 lvl=info msg="Config overridden from Environment variable" logger=settings var="GF_PLUGINS_ALLOW_LOADING_UNSIGNED_PLUGINS=doitintl-bigquery-datasource"
t=2021-06-09T13:45:23+0000 lvl=warn msg="Running an unsigned plugin" logger=plugins pluginID=doitintl-bigquery-datasource pluginDir=/var/lib/grafana/plugins/doitintl-bigquery-datasource/dist
t=2021-06-09T13:45:23+0000 lvl=info msg="Registering plugin" logger=plugins id=doitintl-bigquery-datasource
```

A bit weird that it logs such an old data source, even though https://grafana.com/grafana/plugins/doitintl-bigquery-datasource/ says it's at version 2.0.1.

However it does make the dashboards load fine for now. Would be nice though if it gets signed and this unsecure loading isn't needed.

**Comment by JorritSalverda** [comment]
Given that the `grafanaDependency` property is evaluated using `node-semver` - see https://github.com/npm/node-semver#ranges - it's value should probably be changed to `">=7.0.0"` to get this to work.

**Comment by JorritSalverda** [comment]
With https://github.com/doitintl/bigquery-grafana/pull/344 merged i think this can be closed. When can we expect a `2.0.3` release?

**Comment by ofir5300** [comment]
Hi @JorritSalverda, It is hard to tell when to expect 2.0.3 (I had a major delays with Grafana approvers on v2.0.2...)

**Comment by briantopping** [comment]
Is it worth leaving this open until 2.0.3 is released so people can get notified? Apologies if that's not the way things work around here, I'm new....

**Comment by ofir5300** [comment]
@briantopping eventually this update was included in v2.0.2 release
